frontend/src/pages/EventSignInPage/components/EventSignInForm/index.js

```diff
@@ -10,6 +10,10 @@ import styles from './styles';
 
 import { MajorDropdownField, AffiliateDropdownField } from '@SharedComponents';
 
+import { ButtonWithConfirmationModal } from '@SharedComponents';
```

Comment:
`<ButtonWithConfirmationModal` comes from `@SharedComponents` so you can put it in the curly braces on line 11 as well